src/SimulationRunner.cc

```diff
@@ -122,6 +122,12 @@ SimulationRunner::SimulationRunner(const sdf::World *_world,
         static_cast<int>(this->stepSize.count() / this->desiredRtf));
   }
 
+  // Epoch
+  // TODO: Decide on chrono precision.
```

Comment:
cpplint wants a user assigned to each `TODO`

~~~
Missing username in TODO; it should look like "// TODO(my_username): Stuff."  [readability/todo] [2]
~~~

Reply:
Yeah, I was hoping to decide on this before merge. In fact, seeing how `updatePeriod` uses nanosecond precision, I'll likely change this to the same precision and remove the TODO. That should also get CI to run a bit further.